src/helm/proxy/clients/http_model_client.py

```diff
@@ -32,7 +33,7 @@ def __init__(
         do_cache: bool = False,
     ):
         self.cache: Optional[Cache] = Cache(cache_config) if do_cache else None
-        self.base_url = base_url
+        self.base_url = base_url if not os.environ["HELM_BASE_URL"] else os.environ["HELM_BASE_URL"]
```

Comment:
Could you use a more tightly scoped name e.g. `HELM_HTTP_MODEL_BASE_URL`? Otherwise, LGTM.